app/uk/gov/hmrc/plasticpackagingtax/registration/views/registration_page.scala.html

```diff
@@ -68,7 +68,8 @@ <h2 class="app-task-list__section">
 
         <li>
             @task(1, "registrationPage.organisationDetails", "registrationPage.businessInfo",
-                pptRoutes.HonestyDeclarationController.displayPage(), TaskStatus.Completed)
+                pptRoutes.HonestyDeclarationController.displayPage(),
+            {if (registration.incorpJourneyId.nonEmpty) {TaskStatus.Completed} else {TaskStatus.NotStarted}})
```

Comment:
This reminds me that we need to have a discussion about out FE model.
It would be good to have the Business Details object with an `incorpJourneyId` property that is none optional.

Let's talk today or tomorrow at some point

Reply:
yeap, cool, I've done this for now, while we don't have that object, we'll probably add it once we build check your answers page, lets discuss later.